### DISCUSSION TITLE
Pressing continue on the first form of the Student Developer Pack form does nothing.

### DISCUSSION THREAD

**Discussion by thedev0ps** [community/community#72881] [GitHub Education]
### Hi everyone,

I am trying to apply for the Student Developer Pack, but I don't have a school E-mail address. I filled out everything including my personal email address but every time I press continue I just says "We require applicants to use a school-issued email address to apply."

![image](https://github.com/community/community/assets/74203866/bc86178b-f333-4e06-ba6c-747df8a65aa9)

Any answers appreciated

**Comment by spencersolberg** [comment]
I'm also getting this issue now. Not sure what's gone wrong with my account but I noticed my CoPilot access has been revoked so I went to reapply and it seems the whole system has changed.

**Comment by Slax38** [comment]
👍

**Comment by 4lexxe** [comment]
To access the benefits of the GitHub Education Developer Pack, it is essential to have an institutional email address provided by your university or school. Additionally, it's important that your educational institution is affiliated with this program for you to be eligible. When you try to sign up without these requirements, the system will not allow you to proceed. Once you have all the necessary details, you can apply and gain access to these benefits, which typically takes around a week to be approved. Regards!

**Reply by thedev0ps** [reply]
But I was just reading the FAQs and it said that I can use my personal Email but I have to fill out other information about my school.

**Reply by thedev0ps** [reply]
![image](https://github.com/community/community/assets/74203866/2ed6d4f1-42ea-4e34-bcea-f3525a63a756)

Here it says **SELECTING** not required

**Reply by thedev0ps** [reply]
![image](https://github.com/community/community/assets/74203866/932ebabb-6a72-4472-9d16-2a93ee97d1ff)

Link to page: https://github.com/orgs/community/discussions/17814
